rasa/nlu/featurizers/sparse_featurizer/regex_featurizer.py

```diff
@@ -159,21 +279,54 @@ def load(
         cached_component: Optional["RegexFeaturizer"] = None,
         **kwargs: Any,
     ) -> "RegexFeaturizer":
+        """Load a previously trained component.
+
+        Args:
+            meta: Configuration of trained component.
+            model_dir: Path where trained pipeline is stored.
+            model_metadata: Metadata for the trained pipeline.
+            cached_component: Previously cached component(if any).
+            **kwargs:
+        """
+        finetune_mode = kwargs.pop("finetune_mode", False)
```

Comment:
This is subject to change based on how https://github.com/RasaHQ/rasa/issues/7329 is fixed.